dev/doc/build-system.dune.md

```diff
@@ -3,6 +3,16 @@ Dune-based build system. If you want to enhance the build system
 itself (or are curious about its implementation details), see
 build-system.dev.txt, and in particular its initial HISTORY section.
 
+Quick Start
+===========
+
+You need Dune >= 1.2.1 ; just type `dune build` to build the base Coq
+libraries. No `./configure` step is needed.
```

Comment:
You should add a mention that the tree should not contain left-over compilation outputs. BTW shouldn't you just advise using `make -f Makefile.dune world` instead?

Reply:
Update thanks. Hopefully the need to use make can go away soon. `dune build` should suffice, the makefile is here more as documentation than as a useful tool.